backend/climateconnect_main/settings.py

```diff
@@ -270,9 +270,9 @@
 
 # For Celery we use Redis as a broker URL
 CELERY_BROKER_URL = env('CELERY_BROKER_URL')
-CELERY_BROKER_USE_SSL = {
-    'ssl_cert_reqs': ssl.CERT_REQUIRED
-}
+# CELERY_BROKER_USE_SSL = {
+#     'ssl_cert_reqs': ssl.CERT_REQUIRED
+# }
```

Comment:
Did you just comment these out for development or is this supposed to stay commented out?

Reply:
Yeah, locally you don't have ssl cert so I commented it out.

Reply:
> > The code works perfectly for me! :) I made some comments about how I think the reminders should work, especially that I'd only send them for private notifications. Also the email arrived in my promotions tab on gmail. Likely that is because [noreply@climateconnect.earth](mailto:noreply@climateconnect.earth) is already marked as an email that goes to promotions tab. Therefore it would probably be nicer to add some styling and use a template since removing the styling doesn't seem to make a difference
> 
> I am getting email on my primary setting. I think removing styling worked. But we can use another email like an alias for @tobiasrehm or @positiveimpact's email. Example: `tobias@climateconnect.earth` or `chris@climateconnect.earth`

Ah ok, that's nice! Then sure, we can leave it like this!